README.md

```diff
@@ -148,37 +148,58 @@ E2E/Integration tests are available using [Cypress](https://www.cypress.io/).
 
 #### Requirements:
 
-These files required to run integration tests.
+To run the integration tests locally, you'll need the following files set up:
 - `frontend/.env.local`
-- `frontend/cypress/.env.e2e`
 - `/etc/hosts`
-
+- `frontend/cypress.env.json`
+  - only required if running with Okta
+  
 ```
-# frontend/cypress/.env.e2e
-REACT_APP_BASE_URL=http://localhost.simplereport.gov
-REACT_APP_BACKEND_URL=http://localhost.simplereport.gov/api
-REACT_APP_OKTA_ENABLED=true
-REACT_APP_OKTA_URL=http://localhost:8088
+# /etc/hosts
+# add the following line if running with Okta
+127.0.0.1 localhost.simplereport.gov
 ```
 
-The `frontend/.env.local` file has a template at `frontend/cypress/.env.e2e.sample`. Reach out to another developer to get proper values for these secrets.
-
 ```
-# /etc/hosts
-# add the following line
-127.0.0.1 localhost.simplereport.gov
+# frontend/cypresss.env.json
```

Comment:
nit: the file name looks like it has an extra "s" in cypress

Reply:
Also, just a heads up whenever we upgrade to v10.0.0 of cypress looks like they've gotten rid of support for the `.json` file for [cypress configurations](https://docs.cypress.io/guides/guides/environment-variables#Setting)
<img width="693" alt="Screen Shot 2022-07-11 at 2 05 12 PM" src="https://user-images.githubusercontent.com/20211771/178340207-06c9dcfe-2cfb-47cf-b5af-ac5b52bc5a33.png">

Reply:
that's correct, but the config file is different from [the env file](https://docs.cypress.io/guides/guides/environment-variables#Option-2-cypress-env-json), so I think we should be good

you can set env variables in the config file, but common practice seems to be for the config file to be for universal config things that would be included in the repo and for env to hold passwords and variables specific to local development

Reply:
oooo I see! thank you for the explanation 😸